.travis.yml

```diff
@@ -11,14 +11,26 @@ sudo: false
 
 cache:
   directories:
-    - $HOME/.composer/cache/files
+    - $HOME/.composer/cache
```

Comment:
this needs to be reverted, we only want to cache the dist zips, not the repository information.